operators/src/util/gdal.rs

```diff
@@ -62,6 +62,8 @@ pub fn create_ndvi_meta_data() -> GdalMetaDataRegular {
             spatial_reference: SpatialReference::epsg_4326().into(),
             measurement: Measurement::Unitless,
             no_data_value,
+            time: None,
+            bbox: None,
```

Comment:
Isn't this like global and some months in 2014?

Reply:
340722922944cf22212ccfb50e739e2cb69ca298